test/getApp.test.js

```diff
@@ -0,0 +1,71 @@
+const {getBearerToken} = require('../src/app/auth')
+const faker = require('faker')
+const sinon = require("sinon")
+const axios = require('axios')
+
+let req
+let resSpy = {}
+let authToken
+let nextSpy
+let serverUrl = 'http://localhost:3000/app'
+let headers
+
+
+describe('Retrieve app details - Integration Tests', () => {
+	beforeAll(() => {
+		req = {method: 'GET', query: {}, headers: {}}
+		resSpy.send = sinon.spy()
+		resSpy.status = sinon.spy()
+		nextSpy = sinon.spy()
+		headers = {}
+	})
+
+	describe('Get App details authorization tests', () => {
```

Comment:
Inside 'Retrieve app details - Integration Test' you have 'Get App details authorization tests', which pretty much means the same. 
I would re-name 'Get App details authorization tests'  to just 'authorization', because it's already a part of 'Retrieve app details - Integration Test'.
Also, I would remove the 'tests' from each of the inner describes name, it's redundant.

So when you look at the test results you'll see:
-- Retrieve app details - Integration Test
---- authorisation
---- get by app id

Which I find clearer then:
-- Retrieve app details - Integration Tests
---- Get App details authorization tests
---- Find app by id tests